service/service.go

```diff
@@ -0,0 +1,25 @@
+package service
+
+import (
+	"fmt"
+)
+
+type Service struct {
+	ch chan string
+}
+
+func NewService(ch chan string) *Service {
+	return &Service{ch: ch}
+}
+
+func (s *Service) SetUpstream(host string) {
+	s.ch <- fmt.Sprintf(`kong.response.set_upstream:%s`, host)
+	_ = <- s.ch
+}
+
+func (s *Service) SetTarget(host string, port int) {
+	s.ch <- fmt.Sprintf(`kong.response.set_upstream:["%s", "%d"]`, host, port)
```

Comment:
```suggestion
	s.ch <- fmt.Sprintf(`kong.response.set_target:["%s", %d]`, host, port)
```
(JSON is fine with heterogeneous arrays and [this](https://github.com/Kong/kong-private/pull/166/files#diff-f5f5b7d0be52451e00cc5cb840cfa3fbR274) expects a string and a number)